libmscore/stafftype.cpp

```diff
@@ -936,6 +936,21 @@ void TabDurationSymbol::layout()
                   // once horiz. positions of chords are known
                   }
             }
+      
+      if (_tab->xmlName() == "tabShamisen" && _tab->durationFontName() == "MuseScore Tab 三味線") {
```

Comment:
Unicode in C++ code? We tried to avoid it as much as possible, e.g. for flat, sharp and natural

Reply:
I'll modify it